internal/dataServer/dataServer.go

```diff
@@ -121,7 +122,8 @@ func processLog(reader ngninxLogReader, ip net.IP) (msg, error) {
 		}).Warn("error getting http_x_forwarded_for from the access.log")
 		return msg{}, nil
 	}
-	return msg{SrcIP: ra, DstIP: ip.String(), HTTPStatus: s, XFwdFor: x}, nil
+	p, err := rec.Field("rand_host")
+	return msg{SrcIP: ra, DstIP: ip.String(), HTTPStatus: s, XFwdFor: x, HostProxy: p}, nil
```

Comment:
make sure to update dataServer_test.go to include tests for having and not having p be zero value

Reply:
Added test cases when we have a host_proxy and when the host_proxy filed is empty